test/e2e/common/util_mayastor_crds.go

```diff
@@ -297,10 +297,7 @@ func GetNexusState(uuid string) (string, error) {
 // A volume is published if the "targetNodes" field exists in the MSV.
 func IsVolumePublished(uuid string) bool {
 	_, err := getMsvFieldValue(uuid, "status", "targetNodes")
-	if err != nil {
-		return false
-	}
-	return true
+	return err != nil
```

Comment:
I think you have inverted the logic here.

Reply:
Good spot! fixed.